### PR TITLE
Allow to use remote RPC

### PR DESCRIPTION
**Pull request by splix**

### CODE REVIEW

File: src/lib/networks.js [hunk 1]
```diff
@@ -1,29 +1,17 @@
...
-}
-
-export function getById(id) {
-    if (id === 61) {
-        return Networks[0]
-    }
-    if (id === 62 ) {
-        return Networks[1]
+export function findNetworkDetails(name, chainId, type) {
+    let nameClean = (name || "_unknown_").toLowerCase();
```
Comment (elaineo) [inline src/lib/networks.js]: `(name || "_unknown_").toLowerCase is not a function`

Comment (splix) [inline src/lib/networks.js]: @elaineo fixed, thanks

**Comment by elaineo** [comment]
screen shot 2017-06-27 at 1 12 28 pm

**Comment by splix** [comment]
It should update actually, but after backend will stop existing geth

**Pull request merged**